.travis.yml

```diff
@@ -4,10 +4,14 @@ addons:
   apt:
     sources:
       - deadsnakes
+      - ubuntu-toolchain-r-test
+      - sourceline: 'deb http://archive.ubuntu.com/ubuntu trusty-backports main restricted universe multiverse'
```

Comment:
My Travis-fu is weak, but I couldn't find trusty-backports in [the whitelist](https://github.com/travis-ci/apt-source-whitelist/blob/master/ubuntu.json) (I guess that's what you're referring to?).  There may be another way to get a swig 3, but this seemed like a reasonable one.